dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/validation/hooks/TrackedEntityAttributeValidationHook.java

```diff
@@ -157,7 +156,15 @@ protected void validateAttributes( ValidationErrorReporter reporter,
                 // TODO: Is this really correct? This check was not here
                 // originally
                 // Enrollment attr check fails on null so why not here too?
-                addError( reporter, E1076, attribute );
+                Optional<TrackedEntityTypeAttribute> optionalTea = Optional.ofNullable( trackedEntityType )
```

Comment:
TrackedEntityType cannot be null at this point.
It could be better to use checkNotNull and Optional.of